cypher-shell/src/main/java/org/neo4j/shell/state/BoltStateHandler.java

```diff
@@ -301,6 +302,38 @@ public void changePassword(@Nonnull ConnectionConfig connectionConfig) {
         }
     }
 
+    public void runSingleCommand( @Nonnull ConnectionConfig connectionConfig, @Nonnull String command, Value parameters ) {
+        if (isConnected()) {
+            silentDisconnect();
+        }
+
+        final AuthToken authToken = AuthTokens.basic(connectionConfig.username(), connectionConfig.password());
+
+        try {
+            driver = getDriver(connectionConfig, authToken);
+
+            SessionConfig.Builder builder = SessionConfig.builder()
+                    .withDefaultAccessMode(AccessMode.WRITE)
+                    .withDatabase(SYSTEM_DB_NAME);
```

Comment:
Isn't this a bit weird? I think we will end up here, using the system-db, even if the user invoked it like this: `cypher-shell -d my-db-which-is-not-the-system-db -u user -p pass 'ALTER CURRENT USER SET PASSWORD from "pass" to "newPass"  ';`

Also it is weird to have such connection things going on here, which usually happens in `Main`. Can we not use the same code (i.e. `runUntilEnd`), but parameterize `connectMaybeInteractively` somehow?